tasks/configure.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Configure New Relic PHP agent
+  become: yes
+  ini_file:
+    dest: "{{ sansible_newrelic_agent_php_ini_paths_base }}"
+    mode: 0644
+    section: "{{ item.section | default(None) }}"
+    option: "{{ item.option }}"
+    value: "{{ item.value }}"
```

Comment:
Due to the ongoing Ansible boolean issues, may I recommend:

```yaml
    value: "{% if item.value == true %}true{% elif item.value == false %}false{% else %}{{ item.value }}{% endif %}"
```

Reply:
Good idea, done and done